docs/core-concepts/bot-sequences/actions.mdx

```diff
@@ -1,9 +1,624 @@
+:::
+
+## Action Types
+
+There are many different types of actions that can be used in a bot segment, each useful in their own way.
+The list of supported types are as follows:
+
+* `InputPlayback`: This action plays back a sequence of mouse and keyboard inputs.
+* `Behaviour`: This action runs a MonoBehaviour for a given amount of time.
+* `MonkeyBot`: This action runs a chaos monkey bot for a given amount of time.
+* `Mouse_CVImage`: This action clicks on a region of the screen based on a template image found using computer vision.
+* `Mouse_CVText`: This action clicks on a region of the screen based on text found using computer vision.
+* `Mouse_ObjectDetection`: This action clicks on a region of the screen based on an object found using computer vision.
+* `RestartGame`: This action restarts the game.
+* `QuitGame`: This action quits the game.
+
+### InputPlayback
+
+The `InputPlayback` action type plays back a sequence of mouse and keyboard inputs. This can be useful for
+replaying a sequence of static actions that were previously recorded as part of a play through.
+
+**Data Format**
+
+`type` is set to `InputPlayback`.
+
+```json
+// The `botAction.data` format
+{
+    "startTime": 0.0, // The time offset at which the action should be performed from the start of the segment
+    "inputData": {
+        "mouse": [ // Any number of mouse inputs can be provided
+            {
+                "startTime": ..., // A number indicating the time offset at which the action should be performed from the start of the segment (in seconds)
+                "screenSize": {"x": ..., "y": ...}, // A width and height indicating the size of the screen, which is used to scale the position of the mouse when the screen size is different from the screen size at the time the inputs were recorded
+                "position": {"x": ..., "y": ...}, // A coordinate indicating the position to click, where X and Y are integers
+                "worldPosition": null, // An optional coordinate indicating the position to click in world space, which is often null.
+                "leftButton": ..., // A boolean indicating whether the left mouse button should be pressed
+                "middleButton": ..., // A boolean indicating whether the middle mouse button should be pressed
+                "rightButton": ..., // A boolean indicating whether the right mouse button should be pressed
+                "forwardButton": ..., // A boolean indicating whether the forward mouse button should be pressed
+                "backButton": ..., // A boolean indicating whether the back mouse button should be pressed
+                "scroll": {"x": ..., "y": ...}, // An optional scroll amount, where X and Y are integers
+                "clickedObjectNormalizedPaths": [...], // An optional list of normalized paths to the objects that were clicked
```

Comment:
We should add to the comment that these are used to adjust click coordinates when there are variances in resolution scaling or object position from run to run